extension/src/popup/views/GrantAccess.tsx

```diff
@@ -79,17 +89,18 @@ export const GrantAccess = () => {
       </WarningMessage>
       <PunycodedDomain domain={domain} />
       <SubheaderEl>This website wants to know your public key:</SubheaderEl>
+      <PublicKeyEl>&#9675; {truncatedPublicKey(publicKey)}</PublicKeyEl>
```

Comment:
We have a future ticket to replace the bullet with an identicon. `&#9675;` is a placeholder for now